packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -21,69 +35,373 @@ import styled from 'styled-components';
+                })}
+              </Button>
+            </Flex>
+          }
+        />
+      </Box>
+      {children}
+    </Main>
+  );
+};
+/* -------------------------------------------------------------------------------------------------
+ * ReleaseDetailsBody
+ * -----------------------------------------------------------------------------------------------*/
+const ReleaseDetailsBody = () => {
+  const { formatMessage } = useIntl();
+  const { releaseId } = useParams<{ releaseId: string }>();
+  const [{ query }] = useQueryParams<GetReleaseActionsQueryParams>();
+
+  const { isLoading, isFetching, isError, data } = useGetReleaseActionsQuery({
+    ...query,
+    releaseId,
+  });
+
+  if (isLoading) {
+    return (
+      <ContentLayout>
+        <LoadingIndicatorPage />
+      </ContentLayout>
+    );
+  }
+
+  if (isError) {
+    return (
+      <ContentLayout>
+        <AnErrorOccurred />
+      </ContentLayout>
+    );
+  }
+
+  const releaseActions = data?.data;
+  const releaseMeta = data?.meta;
+
+  if (!releaseActions || !releaseActions.length) {
+    return (
+      <ContentLayout>
+        <EmptyStateLayout
+          content={formatMessage({
+            id: 'content-releases.pages.Details.empty-state.content',
+            defaultMessage: 'This release is empty.',
+          })}
+          icon={<EmptyDocuments width="10rem" />}
+        />
+      </ContentLayout>
+    );
+  }
+
+  return (
+    <ContentLayout>
+      <Flex gap={4} direction="column" alignItems="stretch">
+        <Table.Root
+          rows={releaseActions.map((item) => ({
+            ...item,
+            id: Number(item.entry.id),
+          }))}
+          colCount={releaseActions.length}
+          isLoading={isLoading}
+          isFetching={isFetching}
+        >
+          <Table.Content>
+            <Table.Head>
+              <Table.HeaderCell
+                fieldSchemaType="string"
+                label={formatMessage({
+                  id: 'content-releases.page.ReleaseDetails.table.header.label.name',
+                  defaultMessage: 'name',
+                })}
+                name="name"
+              />
+              <Table.HeaderCell
+                fieldSchemaType="string"
+                label={formatMessage({
+                  id: 'content-releases.page.ReleaseDetails.table.header.label.locale',
+                  defaultMessage: 'locale',
+                })}
+                name="locale"
+              />
+              <Table.HeaderCell
+                fieldSchemaType="string"
+                label={formatMessage({
+                  id: 'content-releases.page.ReleaseDetails.table.header.label.content-type',
+                  defaultMessage: 'content-type',
+                })}
+                name="content-type"
+              />
+            </Table.Head>
+            <Table.LoadingBody />
+            <Table.Body>
+              {releaseActions.map(({ entry }) => (
+                <Tr key={entry.id}>
```

Comment:
Here you should use the releaseAction's id and not the entry's id, because two entries could have the same id if they are from different content types (At least for v4)